File: app/src/main/java/com/islam/retrofitsoap/webservice/request/simplexml/RequestModel.java
package com.islam.retrofitsoap.webservice.request.simplexml;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Element;


public class RequestModel {
    @Attribute(name = "xmlns")
    public String cityNameAttribute;

    @Element(name = "theCityName", required = false)
    public String theCityName;

}

File: app/src/main/java/com/islam/retrofitsoap/webservice/request/tikxml/RequestModel.kt
package com.islam.retrofitsoap.webservice.request.tikxml

import com.tickaroo.tikxml.annotation.Attribute
import com.tickaroo.tikxml.annotation.PropertyElement
import com.tickaroo.tikxml.annotation.Xml

@Xml
class RequestModel {
    @Attribute(name = "xmlns")
    var cityNameAttribute: String? = null
    @PropertyElement(name = "theCityName")
    var theCityName: String? = null
}